content/help/ops/traffic-management/deploy-guidelines/index.md

```diff
@@ -0,0 +1,205 @@
+---
+title: Deployment and Configuration Guidelines
```

Comment:
Title should only have a leading Capital -> See: https://preliminary.istio.io/about/contribute/style-guide/#only-capitalize-the-first-letter-of-headings

Reply:
ignore